awx/main/models/notifications.py

```diff
@@ -233,14 +237,20 @@ def send_notification_templates(self, status_str):
         if notification_templates:
             if status_str == 'succeeded':
                 notification_template_type = 'success'
+            elif status_str == 'running':
+                notification_template_type = 'started'
             else:
                 notification_template_type = 'error'
-            all_notification_templates = set(notification_templates.get(notification_template_type, []) + notification_templates.get('any', []))
-            if len(all_notification_templates):
```

Comment:
why did you remove `if len(all_notification_templates):`, the check for the no-op case? I certainly wouldn't want methods getting added to the on_commit list that don't do anything.

Reply:
Oops! I 🔪 too much!!